docs/how-to/companion-node-types.md

```diff
@@ -5,77 +5,98 @@ description: Learn about the available node types in IPFS Companion.
 
 # Understand node types in IPFS Companion
 
-There are four available node types in IPFS Companion:
+IPFS Companion's preferences screen allows you to choose from several different node types. The full list of available types you'll see in your Companion preferences depends on the browser you're using (i.e. Firefox, Chrome, Brave), but the full list is as follows:
```

Comment:
Agree that this is the right approach for a list of four things, but this will expand over time - @lidel's use of [toc] will keep this automatically updated 😊